lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart

```diff
@@ -12,20 +9,35 @@ import 'package:talawa/view_model/base_view_model.dart';
 /// * `getCurrentOrgUsersList` : to get all users of current organization.
 class SelectContactViewModel extends BaseModel {
   late OrganizationService _organizationService;
+
+  /// orgMembersList is used to store all users of current organization.
   late List<User> orgMembersList = [];
 
-  // initialisation
+  /// This function initializes the [SelectContactViewModel] class.
+  ///
+  /// more_info_if_required
```

Comment:
This line should have been removed from the doc.

Reply:
Sir can you please confirm if you are talking about line no. 17,18

Reply:
18